### DISCUSSION TITLE
Urgent, I need help to debug

### DISCUSSION THREAD

**Discussion by lenamtl** [PHPMailer/PHPMailer#2681] [Q&A]
Hi,

I'm using PHPMailer and it work fine except for 2 scenario randomly.
I'm using a dedicated server with IIS 8
I'm using the latest PHPMail version and I also tested with an older version from last year.

I'm checking the result using  `$mail->send()`
and the email was sent 

So how can I check on the server what happening after the sent why this not really sent like I said this happens only for 2 scenario.
Other scenarios are sending email ok using the same method.
Note when testeing locally it's work fine for all scenario.

Can this be to a PHP ini settings, can you recommend some settings?
Is there a way to get some log in windows or IIS after email is sent?

I only sent a few test so I did reach the daily limits and I can send email using other scenario.
I have tried different email to receive and I also checked junk mail and the mail server.

I'm out to idea and this is a live server so I really appreciate any help.

Thanks

**Comment by Synchro** [comment]
Where is your code? What's in your mail server logs? What happened when you tried the suggestions in the troubleshooting guide?

**Comment by lenamtl** [comment]
Hi,

Thanks I did find the troubleshooting guide https://github.com/PHPMailer/PHPMailer/wiki/Troubleshooting

During my test I found that when this piece of the code is not present the email is not sent ...
This not make sense I know, but when this code is present this is work fine.
```
if( !$mail->send() ) {
	exit;
}
```

Note 
I'm using this method
```
use PHPMailer\PHPMailer\Exception;
use PHPMailer\PHPMailer\PHPMailer;

require dirname(__FILE__)."/vendor/phpmailer/src/Exception.php";
require dirname(__FILE__)."/vendor/phpmailer/src/PHPMailer.php";	
require dirname(__FILE__)."/vendor/phpmailer/src/SMTP.php";

```

**Reply by Synchro** [reply]
If you don't call `send`, your message will not be sent. It's not very complicated! Do you have an actual problem? With code, error messages, and logs?

**Comment by lenamtl** [comment]
You can close this for now.
Thanks
